geps/gep-1709.md

```diff
@@ -12,32 +12,24 @@ the Gateway API project and receive recognition (e.g. badges).
 
 ## TODO
 
-The following are items that we intend to resolve before we consider this GEP
-`implementable`:
-
-- We still need to sort out how we're going to display conformance results.
-  Badges are well and good, but we also want some kind of summarized
-  display of any given implementation's conformance results somewhere on
-  our upstream website (so far the thought has been the [implementations
-  page][impl]). We should look for prior art in Kubernetes SIGs and also
-  reach out to some frontend developers to come up with something really
-  solid prior to moving to `implementable`.
+The following are items that **MUST** be resolved before we move past the
+`provisioning/prototyping` status:
+
 - We need to think about how we're going to highlight and report on the
   results for `experimental` features, and make sure this is explicitly
   written out.
-- We are [gathering feedback from SIG Arch][sig-arch-feedback]. We need to let
-  this soak for some time, bring up the topic at SIG Arch community meetings, and
-  generally make changes according to their feedback before considering this
-  `implementable`.
-- Right now we've said that `GatewayClass` and `Gateway` tests will be
-  implicitly added to testing profiles were they are referenced, but we're
-  not sure yet whether this is going to work out for `ReferenceGrant` as we
-  know of at least one implementation that explicitly doesn't support it.
```

Comment:
Why was this one removed?

Reply:
Check the commit history, the commit includes an extended comment which explains: basically we decided to go the other way and implicitly added ReferenceGrant tests to profiles. It seems this might actually be fine for all implementations, if not we technically allow the feature to be disabled.